performance-test/src/main/java/org/apache/mesos/elasticsearch/performancetest/ElasticsearchSchedulerContainer.java

```diff
@@ -0,0 +1,41 @@
+package org.apache.mesos.elasticsearch.performancetest;
+
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.command.CreateContainerCmd;
+import org.apache.mesos.mini.container.AbstractContainer;
+
+import java.security.SecureRandom;
+import java.util.stream.IntStream;
+
+/**
+ * Container for the Elasticsearch scheduler
+ */
+@SuppressWarnings({"PMD.AvoidUsingHardCodedIP"})
+public class ElasticsearchSchedulerContainer extends AbstractContainer {
```

Comment:
Fairly noted. Indeed, there's a lot of overlap with systemTest. My intention was to get it out as soon as possible for getting feedback, since framework was very actively developed, so a lot of things changed rapidly. Once things stabilize, it makes sense to move things under "commons".